android/properties.gradle.kts

```diff
@@ -10,7 +11,7 @@ fun Project.readProperties(file: String): Properties {
       ->
       properties.load(reader)
     }
-  } else println("FILE_NOT_FOUND_EXCEPTION: File $file not found")
+  } else throw FileNotFoundException("File $file not found")
```

Comment:
so far, this hasn't been a requirement, why are we making it a requirement now?

Reply:
this was trying to fix an issue that occurred last time when building the release APKs, if the file wasn't specified correctly it failed silently and dummy credentials were updated meaning the release APK was unusable.

@dubdabasoduba pointed out it fails even in dev mode, i've since patched it to only throw an exception when an external file path is specified - the most likely case for when the aforementioned issue can crop up